packages/core/src/vite-rpc/assets.ts

```diff
@@ -2,6 +2,9 @@ import fsp from 'node:fs/promises'
 import fg from 'fast-glob'
 import { join, resolve } from 'pathe'
 import { imageMeta } from 'image-meta'
+import { BirpcGroupReturn } from 'birpc'
+import type { ViteDevServer } from 'vite'
```

Comment:
We haven't `Vite` dependency in the core package, just let's remove the type import for now.